test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/mq_as_uri_param_for_metadata_nocache.test

```diff
@@ -1,868 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
-#
-# This file is part of Orion Context Broker.
-#
-# Orion Context Broker is free software: you can redistribute it and/or
-# modify it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# Orion Context Broker is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
-# General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
-#
-# For those usages not covered by this license please contact with
-# iot_support at tid dot es
-
-# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
-
---NAME--
-mq as URI param for metadata in filters - no subCache variant
-
---SHELL-INIT--
-dbInit CB
-brokerStart CB 0 IPv4 --noCache
-accumulatorStart --pretty-print
-
---SHELL--
-
-#
-# The only difference regarding the cache version of this test (mq_as_uri_param_for_metadata.test)
-# is in step 05b. This test doesn't include lastSuccess and lastSucessInfo fields, while
-# mq_as_uri_param_for_metadata.test does
-#
-# In step 05 there is a race condition between the creation of the csub document
-# at DB and the notification being sent. If the notification being created is the one that wins,
-# the csub document doesn't exist at the DB when it tries to update the lastSuccess and
-# lastSucessInfo. The grep -v in step 05b is for skipping these elements and make the checking
-# valir for both cases. The REGEX in Content-Lenght covers both cases.
```

Comment:
This is no longer a difference regarding to test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/q_and_mq_as_uri_param_for_metadata.test since the removal of initial notifications. Thus, this file has been removed and test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/q_and_mq_as_uri_param_for_metadata.test used for both cases (`brokerStart CB 0 IPv4 --cache` -> `brokerStart CB 0 IPv4`)

NTC (informative)